go-controller/hybrid-overlay/pkg/types/types.go

```diff
@@ -11,9 +11,13 @@ const (
 	HybridOverlayNodeSubnet = HybridOverlayAnnotationBase + "node-subnet"
 	// HybridOverlayDRMAC holds the MAC address of the Distributed Router/gateway
 	HybridOverlayDRMAC = HybridOverlayAnnotationBase + "distributed-router-gateway-mac"
+	// HybridOverlayExternalGW holds the IP address of the external gateway to route default traffic
```

Comment:
@rcarrillocruz this should be explicit that it's on the *Namespace* object, not the Node. We should document where these annotations apply at least.